frontend/src/api/auth.js

```diff
@@ -6,11 +6,11 @@ import { useGlobal } from '../state';
 
 const getTokenFromQuery = () => {
   const queryValues = qs.parse(window.location.search);
-  return queryValues.token;
+  return queryValues.code;
```

Comment:
Changed to .code because there seems it gives as per github authO it gives us a code, which needs to be posted through endpoint to give us a token. So, .token was giving undefined.

Reply:
This is interesting. I think it would be revisited later.